### ISSUE TITLE
Value data from feeds/add/{data} doesn't passed directly to input where it was previously added

### ISSUE THREAD

**Issue by dancetrain** [hplush/slowreader#130]
Steps to reproduce:
 - Open http://localhost:5173/feeds/add/
 - Add value to `Web page URL or social account handle` input field (example: https://soundcloud.com/redaxes/flawless-club-mix-feat-echo)
 - Observe how URL is changed regardless of errors (example: http://localhost:5173/feeds/add/https%3A%2F%2Fsoundcloud.com%2Fredaxes%2Fflawless-club-mix-feat-echo)
 - Refresh the page - value from URL is not returned to input box

**Comment by kmpeeduwee** [comment]
I want to take it

**Comment by ai** [comment]
Sure, it is your.

**Comment by kmpeeduwee** [comment]
@ai Hi! I found how to implement this. But something tells me that this is not quite the correct way to add this code to onMount in preview.ts. Can you comment on this approach?

```
onMount($candidates, () => {
  let page = router.get()
  page.route === 'add' && setPreviewUrl(page.params.url!)
  return $links.listen(() => {})
})
```

**Comment by ai** [comment]
Looks normal in terms of API. But I don’t understand why we have `$links.listen(() => {})`

**Comment by kmpeeduwee** [comment]
@ai I apologize, but this was due to my not understanding the code well enough; it was a piece of old code. I propose this version, in which I subscribe to the router and return the unsubscribe function:
```js
onMount($candidates, () => {
  let unsubscribeRouter = router.subscribe(page => {
    if (page.route === 'add') {
      setPreviewUrl(page.params.url!);
    }
  });

  return () => {
    unsubscribeRouter();
  };
});
```

**Comment by ai** [comment]
Fixed 506c628
